File: Disciplina-AlgProg/Exercicios/Exercicios/Lista 1 - Operadores Aritméticos/Atividade_B/Lst01qst13.py
import sys
sys.path.append(".")
from utils import *

def main():
    showPresentation("Leia um valor em real (R$), calcule e escreva 70% deste valor.")
    #Entrada
    value = getNumber("Digite um valor em reais: ")

    #Processamento
    newValue = value * 0.7

    #Saída
    showResult(f"70% de {value} são {newValue}")

main()

File: Disciplina-AlgProg/Exercicios/Exercicios/Lista 1 - Operadores Aritméticos/Atividade_B/Lst01qst13.c
#include <stdio.h>

// 70% de um valor

float GetNumber(char message[]){
    float number;
    printf("%s", message);
    scanf("%f", &number);

    return number;
}

int main(){
    //Entrada
    float value = GetNumber("Digite um numero em reais: ");

    //Processamento
    float newValue = value * 0.7;

    //Saída
    printf("70%% de %.2fR$ equivalem a %.2fR$", value, newValue);

    return 0;
}